### ISSUE TITLE
Unable to handle database name with `tbl()` function

### ISSUE THREAD

**Issue by chezou** [ianmcook/implyr#2]
It would be nice to handle database name with `tbl()` function. In current version of implyr, it returns an error as follows:

```r
> airports <- tbl(impala, "u_ariga.airports_pq")
Error in new_result(connection@ptr, statement) : 
  nanodbc.cpp:1344: HY000: [Cloudera][ImpalaODBC] (110) Error while executing a query in Impala: [HY000] : AnalysisException: Could not resolve table reference: 'u_ariga.airports_pq'
```

**Comment by chezou** [comment]
I didn't dig the route cause with debugging yet, but I guess there is the same problem that sparklyr had.
https://github.com/rstudio/sparklyr/pull/501

**Comment by ianmcook** [comment]
The resolution of this should use forward-compatible syntax. That's not possible at the moment because Hadley and Kirill Müller and others are actively working to solve this problem in dplyr and DBI, and the syntax that will be used has not yet been decided. Closing this for now, but I will revisit it once there is an idiomatic syntax for this established by dyplr.

**Comment by ianmcook** [comment]
This is solved in dplyr 0.7.0 with the new function [`in_schema()`](http://dbplyr.tidyverse.org/reference/in_schema.html).
